pandas/core/frame.py

```diff
@@ -1480,11 +1480,23 @@ def to_stata(self, fname, convert_dates=None, write_index=True,
             format that you want to use for the dates. Options are
             'tc', 'td', 'tm', 'tw', 'th', 'tq', 'ty'. Column can be either a
             number or a name.
+        write_index : bool
+            Write the index to Stata dataset.
         encoding : str
             Default is latin-1. Note that Stata does not support unicode.
         byteorder : str
             Can be ">", "<", "little", or "big". The default is None which uses
             `sys.byteorder`
+        time_stamp : datetime
```

Comment:
Yes.  Arguments were present but uncodumented.  Mentioned in one fo the issues this closes.